api/nodes/views.py

```diff
@@ -232,12 +236,38 @@ def get_default_odm_query(self):
         query = base_query & permission_query
         return query
 
-    # overrides ListCreateAPIView
+    # overrides ListBulkCreateJSONAPIView, BulkUpdateJSONAPIView
     def get_queryset(self):
-        query = self.get_query_from_request()
-        return Node.find(query)
+        # For bulk requests, queryset is formed from request body.
+        if is_bulk_request(self.request):
```

Comment:
The JSON API Spec does not mention any way to do bulk gets, but we could use the format for DELETE (which also does not normally have JSON data). But if the GET can't have json, then there's not much we can do. DELETE can have JSON, even if people don't like it.